### PR TITLE
Add ParmEd

### PR DESCRIPTION
**Pull request by avirshup**
Creates a clone of the ParmEd package from the `omnia` channel

### CODE REVIEW

File: recipes/parmed/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: ParmEd-{{ version }}.tar.gz
+  url: https://github.com/{{ name }}/{{ name }}/archive/{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+
+build:
```
Comment (synapticarbors) [inline recipes/parmed/meta.yaml]: This package has a command line interface as an entry point:

https://github.com/ParmEd/ParmEd/blob/master/setup.py#L17

please add the appropriate entry:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points

File: recipes/parmed/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    - python
+    - numpy
+    - setuptools
+
+  run:
+    - python
+    - numpy
+    - pandas
+
+test:
```
Comment (synapticarbors) [inline recipes/parmed/meta.yaml]: add a test for the entry_point cli:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands

File: recipes/parmed/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  imports:
+    - parmed
+  
+
+about:
+  home: http://parmed.github.io/ParmEd
+  license: LGPLv2+
```
Comment (synapticarbors) [inline recipes/parmed/meta.yaml]: Please add the `license_file: GNU_LGPL_v2`

also change to `license: LGPL-2.1`

File: recipes/parmed/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+
+build:
+  number: 1
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - numpy
```
Comment (synapticarbors) [inline recipes/parmed/meta.yaml]: Is `numpy` required to actually build the package? If not, remove it.

File: recipes/parmed/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    - setuptools
+
+  run:
```
Comment (synapticarbors) [inline recipes/parmed/meta.yaml]: It's not entirely clear to me from looking at the repo what the requirements are.

File: recipes/parmed/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+    - python
+    - numpy
+    - pandas
+
+test:
+  imports:
+    - parmed
```
Comment (synapticarbors) [inline recipes/parmed/meta.yaml]: Maybe add the submodules as well:

https://github.com/ParmEd/ParmEd/blob/master/setup.py#L63-L68

File: recipes/parmed/meta.yaml (hunk 5 above)
Comment (avirshup) [reply recipes/parmed/meta.yaml]: Hmmm ... same here. Would appear that only numpy is strictly required, but I'll run some tests to confirm.

**Comment by avirshup** [comment]
@synapticarbors - thanks, the review and comments are very much appreciated.

I think the issues you've raised should be addressed as of commit 067bf69 - please let me know if I missed anything or if there's anything else I should take care of.

**Pull request closed without merging**